web/src/beta/lib/reearth-ui/components/PopupMenu/index.tsx

```diff
@@ -67,34 +69,65 @@ export const PopupMenu: FC<PopupMenuProps> = ({
     [onOpenChange, open],
   );
 
+  const renderSingleItem = (item: PopupMenuItem, index: number) => {
+    const { icon, id, onClick, path, selected, subItem, title } = item;
+    return (
+      <Item
+        key={index}
+        size={size}
+        onClick={() => {
+          onClick?.(id);
+          handlePopOver(false);
+        }}>
+        {icon && (
+          <Icon icon={icon} size="small" color={iconColor ? iconColor : theme.content.weak} />
+        )}
+        <div
+          style={{
```

Comment:
use styled component to define this styles

Reply:
Done